src/codegen_types/datetime.cr

```diff
@@ -1,11 +1,11 @@
 module AST
   class DateTimePrimitiveType
     def typescript_decode(expr)
-      "moment.utc(#{expr}, \"YYYY-MM-DDTHH:mm:ss.SSS\").toDate()"
+      "new Date(#{expr})"
     end
 
     def typescript_encode(expr)
-      "moment.utc(#{expr}).format(\"YYYY-MM-DDTHH:mm:ss.SSS\")"
+      "#{expr}.toISOString()"
```

Comment:
toISOString() addes "Z" on the end. Current format has no timezone information. Also, the date must first be converted to UTC. This is not a problem for the server (which localtime = utc), but for the targets web and nodeclient this is an issue.

When decoding it needs to be converted from UTC to localtime, as well.